Aandachtspunten.md

```diff
@@ -1,9 +1,28 @@
 ## Aandachtspunten
 
+### Inhoud
+
 - Alle randvoorwaarden zijn voldaan.
 
-- Er wordt gebruik gemaakt van Dependency Injection via de constructor OF via @Autowired
+- Er wordt gebruik gemaakt van Dependency Injection via de constructor (OF via @Autowired, maar dat heeft niet de voorkeur
```

Comment:
De regel met de voorkeur voor Dependency Injection via de constructor en de notatie @Autowired is goed, maar ik zou aanbevelen om de voorkeur weg te laten in de tekst om de leesbaarheid te verbeteren. Bijvoorbeeld: 'Het gebruik van Dependency Injection via de constructor is de aanbevolen werkwijze.'